server/server.go

```diff
@@ -178,9 +179,10 @@ func parseVersionByte(w http.ResponseWriter, r *http.Request) (byte, error) {
 		return 0, fmt.Errorf("version byte is not a single byte: %s", versionByteHex)
 	}
 	switch versionByte[0] {
-	case byte(commitments.CertV0), byte(commitments.CertV1):
-		return versionByte[0], nil
-
+	case byte(commitments.CertV0):
```

Comment:
Alternatively, you could create a `ByteToEigenDACertVersion` function that returns an error if the input byte isn't a valid version. Then it would look something like this:

```
certVersion, err := ByteToEigenDACertVersion(versionByte[0])
if err != nil {
    http.Error(w, fmt.Sprintf("unsupported version byte %x", versionByte), http.StatusBadRequest)
    return 0, fmt.Errorf("unsupported version byte %x", versionByte)
}

return certVersion, nil
```

Up to you if you want to make this change or not.

Reply:
[a94383b](https://github.com/Layr-Labs/eigenda-proxy/pull/379/commits/a94383b1434830577a9c8d7f528cb5a7d2aa5c84)